packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -15,25 +14,18 @@ import { localize } from '../i18next';
 import './advertiser-page.scss';
 
 const RowComponent = React.memo(({ advert, showAdPopup, style }) => {
-    const { advertiser_id } = React.useContext(Dp2pContext);
-    const {
-        account_currency,
-        advertiser_details,
-        counterparty_type,
-        local_currency,
-        max_order_amount_limit_display,
-        min_order_amount_limit_display,
-        price_display,
-    } = advert;
+    const { advertiser_page_store, general_store } = useStores();
+    const { local_currency, max_order_amount_limit_display, min_order_amount_limit_display, price_display } = advert;
 
-    const is_buy_advert = counterparty_type === buy_sell.BUY;
-    const is_my_advert = advertiser_details.id === advertiser_id;
+    const is_buy_advert = advertiser_page_store.counterparty_type === buy_sell.BUY;
+    const is_my_advert = advertiser_page_store.advertiser_details.id === general_store.advertiser_id;
 
     return (
         <div style={style}>
             <Table.Row className='advertiser-page__adverts-table_row'>
                 <Table.Cell>
-                    {min_order_amount_limit_display}&ndash;{max_order_amount_limit_display} {account_currency}
+                    {min_order_amount_limit_display}&ndash;{max_order_amount_limit_display}{' '}
+                    {general_store.client.currency}
```

Comment:
Could be reformatted as below:

```JS
{`${min_order_amount_limit_display}–${max_order_amount_limit_display} {general_store.client.currency}`}
```

Optionally can destructure `currency` to be less verbose.

```JS
const { currency } = general_store.client;
```